### ISSUE TITLE
Weird key error after searching for locations

### ISSUE THREAD

**Issue by marcuspoon2** [gcivil-nyu-org/Team_Team2_CS-GY-6063-Fall2021#173]
Not sure how to recreate this issue, but sometimes after searching for restrooms and clicking to see a location's details quickly, I'll see this KeyError pop up. This has happened a few times while testing your app 

<img width="781" alt="Screen Shot 2021-11-19 at 12 13 18 AM" src="https://user-images.githubusercontent.com/67922103/142568997-996c6357-7723-4084-ad89-7cba9696ed03.png">

**Comment by Scott-Canning** [comment]
I've run into this same key error as Marcus and it also occurred fairly randomly - haven't isolated the trigger and it worked fine 30 seconds later. It happened when I searched for 'tandon' and clicked the first Starbucks location under Unfiltered Results. Probably unrelated but I changed the default for all filters to 'Yes' in my profile prior to this result instance.

![image](https://user-images.githubusercontent.com/34758484/142685616-a06e6d7b-07c5-43c8-a62a-4dc8f4e95bf0.png)

![image](https://user-images.githubusercontent.com/34758484/142685387-65720f10-985c-45c8-877a-46e7936947dc.png)
